explorer/src/lib/services/gql-queries.js

```diff
@@ -124,7 +115,17 @@ export const getTransactionsQueryInfo = (amount) => ({
 
 /** @param {string} id */
 export const getTransactionDetailsQueryInfo = (id) => ({
```

Comment:
Maybe it shoud just be `getTransactionQueryInfo` as we don't have two queries anymore?